### PR TITLE
MMP-200 Reestructure classloader model

### PR DESCRIPTION
**Pull request by WilliamPinho**

### CODE REVIEW

File: mule-artifact-it/src/test/resources/expected-classloader-model-project/META-INF/mule-src/empty-classloader-model-project/log.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[INFO] Error stacktraces are turned on.
```
Comment (damiansima) [inline mule-artifact-it/src/test/resources/expected-classloader-model-project/META-INF/mule-src/empty-classloader-model-project/log.txt]: do we use this file?

File: mule-packager/src/main/java/org/mule/tools/api/classloader/model/ClassLoaderModel.java [hunk 2]
```diff
@@ -58,56 +54,32 @@ public void setDependencies(List<Artifact> dependencies) {
...
     ClassLoaderModel copy = new ClassLoaderModel(version, artifactCoordinates);
     copy.setDependencies(dependenciesCopy);
-    copy.setMulePlugins(mulePluginsCopy);
 
     return copy;
   }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) return true;
```
Comment (damiansima) [inline mule-packager/src/main/java/org/mule/tools/api/classloader/model/ClassLoaderModel.java]: please add {}

File: mule-artifact-it/src/test/resources/expected-classloader-model-project/META-INF/mule-src/empty-classloader-model-project/log.txt (hunk 1 above)
Comment (WilliamPinho) [inline mule-artifact-it/src/test/resources/expected-classloader-model-project/META-INF/mule-src/empty-classloader-model-project/log.txt]: How did you find this in the middle of all these pom files? =D

File: mule-packager/src/main/java/org/mule/tools/api/classloader/model/ClassLoaderModel.java (hunk 2 above)
Comment (WilliamPinho) [inline mule-packager/src/main/java/org/mule/tools/api/classloader/model/ClassLoaderModel.java]: Done!

**Pull request merged**